public/js/main.js

```diff
@@ -63,7 +63,40 @@ async function markIncomplete(){
         const data = await response.json()
         console.log(data)
         location.reload()
-    }catch(err){
+    } catch (err) {
         console.log(err)
     }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-movie').addEventListener('click', getMovie)
+
+async function getMovie() {
+    let givenTitle = document.getElementById('text-movie').value;
+    try {
+        const response = await fetch(`search/?title=${givenTitle}`);
+        const data = await response.json().then(x => x);
+        addMovieDataToDOM(data[0]);
+
```

Comment:
There's currently no way for a user to know if their input is valid bc there's no check for their input. In the `try` block, the code is always fetching and responding even if the input is not valid.

Can we put in `if else` in the `try` here to check if their input can be found?

something like this:
```
try {
 const response = await fetch(`search/?title=${givenTitle}`);
 const data = await response.json().then(x => x);
  if (data.length !== 0) {
   addMovieDataToDOM(data[0]);
  }
  else {
   alert('Media not found. Please try another movie/tv show.');
  }
```

Reply:
Great comment. Addressed in [5b849bf](https://github.com/CadmiumRed40/FindAFlick/pull/24/commits/5b849bf7e84c274d49be9835c5a3998d4dc8fb3a).

Please mark resolved if satisfactory.